Luke_Smith_Portfolio.css

```diff
@@ -0,0 +1,126 @@
+body {aaa
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: auto;
+  background-color: #ffffff;
+}
+
+.topnav a {
+  float: left;
+  color: #000000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  width: auto;
+}
+
+.topnav a:hover {
+  background-color: #dddddd;
+  color: #000000;
+}
+
+body {
+  background: #555555;
+}
+
+/*About Me Section*/
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #000000;
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.column {
+  float: left;
+  width: 33.3%;
+  margin-bottom: 16px;
+  padding: 0 8px;
+}
+
+.about-section {
+  padding: 50px;
+  text-align: center;
+}
+
+.container {
+  padding: 0 16px;
+}
+
+.container::after, .row::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+.title {
+  color: #000000;
+  font-style: italic;
+}
+
+.button {
+  border: none;
+  outline: 0;
+  display: inline-block;
+  padding: 8px;
+  color: #FFFFFF;
+  background-color: #000;
+  text-align: center;
+  cursor: pointer;
+  width: 100%;
+}
+
+.button:hover {
+  background-color: #555;
+}
+
+.maydenlogo {
+	width: 500px;
+}
+
+.maydenlogo img {
+	width: 100%;
+}
+
+/*Contact section links*/
+.contact:link, .contact:visited {
+  background-color: #FFFFFF;
+  color: #000000;
+  border: 2px #000000;
```

Comment:
This border is not appearing. Either:
- change this line to "border: 2px solid #000000" if you want the border to appear around your contact links.
- delete this line if you prefer how it currently looks.